tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -261,14 +261,8 @@ func (d *deployer) updateCluster(yes bool) error {
 func (d *deployer) IsUp() (bool, error) {
 	wait := d.ValidationWait
 	if wait == 0 {
-		if d.TerraformVersion != "" || d.CloudProvider == "digitalocean" {
-			// `--target terraform` doesn't precreate the API DNS records,
-			// so kops is more likely to hit negative TTLs during validation.
-			// Digital Ocean also occasionally takes longer to validate.
-			wait = time.Duration(20) * time.Minute
-		} else {
-			wait = time.Duration(15) * time.Minute
-		}
+		// kOps is more likely to hit negative TTLs for API DNS during validation.
```

Comment:
Maybe we should sleep for a minute between crest cluster --yes and validate cluster?

Reply:
FWIW the prow job shows that the kops cli is finding the placeholder IP immediately during `kops validate cluster`: 

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-kops-e2e-aws-karpenter/1705830841394925568#

but there are occurrences where tests have failed because validation was taking too long (but was on track to eventually succeed) [example](https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/e2e-kops-aws-karpenter/1705616628923764736) so we should probably keep this in but look into speeding up time to pass validation.

Reply:
ref: https://github.com/kubernetes/kops/issues/15961